src/content-layout/__integ__/content-layout.test.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { BasePageObject } from '@cloudscape-design/browser-test-tools/page-objects';
+import useBrowser from '@cloudscape-design/browser-test-tools/use-browser';
+import styles from '../../../lib/components/content-layout/styles.selectors.js';
+
+class ContentLayoutPageObject extends BasePageObject {
+  async isOverlapEnabled() {
+    const backgroundElement = await this.browser.$(`.${styles.background}`);
+    const classes = await backgroundElement.getAttribute('class');
+    return !classes.includes(styles['is-overlap-disabled']);
```

Comment:
Given the behavior of the component the above makes more sense to me, but I think @connorlanigan should weigh in on both of these test threads.

Reply:
I think those two would be equivalent, right?

You currently look for an element with `.background`, and then check if the classes of that element include the class `is-overlap-disabled`. 

Shouldn't that be equivalent to just looking for the selector `.background.is-overlap-disabled`?

Reply:
They are not fully equivalent: if the background element does not render for some reason (e.g. an error on the page), a  merged selector would not detect that, and instead it would fail silently by returning that the overlap is enabled.

Reply:
Ah, okay. So what you're saying is that you need the first line (`await this.browser.$`) to throw in case that element doesn't exist?

Reply:
Yes